### PR TITLE
Update Profiles.adoc

### PR DESCRIPTION
**Pull request by omoz4real**

### CODE REVIEW

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 1]
```diff
@@ -44,35 +44,35 @@ rules of the Java Community Process. Typically, a proposal to create a
 new profile, or to revise an existing one, will be submitted as a Java
 Specification Request (JSR). Once the JSR is approved, an expert group
 will be formed and conduct work as dictated by the process. The JSR for
-a profile must mention the version of the Java EE Platform that it
+a profile must mention the version of the Jakarta EE Platform that it
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc]: This section needs some work.  We are no longer following the JCP processes.  We're following the Jakarta EE Specification Process.  And, we use Jakarta Specification Projects (within Eclipse) to define new specifications (or profiles).

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 2]
```diff
@@ -82,7 +82,7 @@ EE platform or any profile on which it builds, as long as the
 corresponding compatibility requirements are satisfied.
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc]: Missed a Java EE reference...

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 3]
```diff
@@ -96,7 +96,7 @@ EE platform or by any profile on which the present one builds.
 The specification for individual technologies
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc]: Missed another Java EE reference...

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 4]
```diff
@@ -132,53 +132,53 @@ otherwise indicated.
...
 annotations defined by the Common Annotations specification ( _Resource_
 , _Resources_ , _PostConstruct_ , _PreDestroy_ )
 
 The following functionality is required to be
-supported in all Java EE profiles:
+supported in all Jakarta EE profiles:
 
 === JNDI “java:” naming context (see link:#a607[See JNDI Naming Context]”)
 
 * Java Transaction API (JTA)
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc]: Jakarta Transaction

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 5]
```diff
@@ -204,10 +204,10 @@ associated EJB QL, which have been made optional)
 * JSON-B 1.0
 * Concurrency Utilities 1.0
 * Batch 1.0
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc]: All of these technologies should be expanded to the new Jakarta naming convention...  Jakarta Enterprise Beans, Jakarta Servlet, etc

File: specification/src/main/asciidoc/platform/Profiles.adoc [hunk 6]
```diff
@@ -316,8 +316,8 @@ Edition (Java SE) Requirements].”
 === Transactions
 
 Application clients are not required to have
-direct access to the transaction facilities of the Java EE platform. A
-Java EE product is not required to provide a JTA _UserTransaction_
+direct access to the transaction facilities of the Jakarta EE platform. A
+Jakarta EE product is not required to provide a JTA _UserTransaction_
```
Comment (kwsutter) [inline specification/src/main/asciidoc/platform/Profiles.adoc]: JTA == Jakarta Transaction

**Comment by keilw** [comment]
Why is this not ready for merge, does @omoz4real have no ECA or is the "Signed-off by" missing?

**Comment by kazumura** [comment]
> Why is this not ready for merge, does @omoz4real have no ECA or is the "Signed-off by" missing?

Because the Author of the commits is anonymous (omoz4real@****.noreply).

**Comment by omoz4real** [comment]
Updated

**Comment by omoz4real** [comment]
Sorry , I don't understand what you mean by being anonymous. What do I have to do to not to be anonymous. Please, this is my first pull request ever and don't really know how to go about it from here.
Thanks.
    On Sunday, 7 July 2019, 23:52:44 GMT+1, Kenji Kazumura <notifications@github.com> wrote:  
 
 

Why is this not ready for merge, does @omoz4real have no ECA or is the "Signed-off by" missing?


Because the Author of the commits is anonymous (omoz4real@****.noreply).

**Comment by kazumura** [comment]
> Sorry , I don't understand what you mean by being anonymous. What do I have to do to not to be anonymous. Please, this is my first pull request ever and don't really know how to go about it from here. 

When you type  "git log --pretty-full", you can see the "Author" header. It muse be your real mail address.
Typically you should do 
  git commit --amend --author="your  name" <your@real.mail>
or set [user] section in .gitconfig as follows
  email = your@real.mail
  name = your name

**Comment by omoz4real** [comment]
Thanks a lot. Didn't know about this. I am so grateful. 
    On Monday, 8 July 2019, 00:59:14 GMT+1, Kenji Kazumura <notifications@github.com> wrote:  
 
 

Sorry , I don't understand what you mean by being anonymous. What do I have to do to not to be anonymous. Please, this is my first pull request ever and don't really know how to go about it from here.


When you type "git log --pretty-full", you can see the "Author" header. It muse be your real mail address.
Typically you should do
git commit --amend --author="your name" your@real.mail
or set [user] section in .gitconfig as follows
email = your@real.mail
name = your name

File: specification/src/main/asciidoc/platform/Profiles.adoc (hunk 2 above)
Comment (omoz4real) [reply specification/src/main/asciidoc/platform/Profiles.adoc]: Hi Kevin, Sorry I don't quite understand the changes that needs to be made here. Do I have to remove the "EE platform" ? Please I am quite a novice at this.

Thanks

**Pull request closed without merging**